### PR TITLE
Update swagger.yaml

### PR DESCRIPTION
**Pull request by enegron-sl**

### CODE REVIEW

File: spec/swagger.yaml [hunk 1]
```diff
@@ -124,26 +124,20 @@ tags:
...
-      5. Batch execution process waiting until all files are uploaded and added to the Job and then, if was requested, authorizes the Job.
-      6. Clients can check batch status and see if all files uploaded and attached successfully, what errors happened for each file, and statistics for added/skipped strings.
-
-      After Batch is completed it can't be reused.
-
-      **Job batches Limitations**
-      - Service doesn't guarantee correct processing of multiple batches for the same job in parallel. Adding files to a new batch should be done only after previous batch reached "completed" state.
-      - Service doesn't guarantee correct processing of Batch if Job and Files are modified in UI or with another integration.
-
+      The Job Batches APIs provide a wrapper around Jobs and Files that simplifies workin with them. Users can more easily combine uploading files and adding them to Jobs.
```
Comment (hchaps) [inline spec/swagger.yaml]: Fixed typos. Update to: 
The Job Batches API provides a wrapper around Jobs and Files that simplifies working with them.

File: spec/swagger.yaml [hunk 2]
```diff
@@ -124,26 +124,20 @@ tags:
...
+      Generally, adding multiple files to a Job forces the user to implement polling. This is because The Jobs API expects a file to be uploaded and fully processed before being added to a Job. In other words the Add File to Job API is *not* actually an API ~upload~, it just associates a file that was uploaded separately, to the Job. That process of uploading a file to Smartling is asynchronous.  The file needs to be fully processed before it's ready to be worked with, for example, before it's added to a Job. Lastly a Job can't be authorized until any background tasks for its component parts are completed. Job Batches simplifies these complexities.
```
Comment (hchaps) [inline spec/swagger.yaml]: Fixed typos. Update to:
This is because the Jobs API expects...

File: spec/swagger.yaml [hunk 3]
```diff
@@ -124,26 +124,20 @@ tags:
...
+      The Job Batches API utilizes internal Smartling events to optimize the process of adding uploading files and adding them to Jobs.
```
Comment (hchaps) [inline spec/swagger.yaml]: uploading > uploaded

File: spec/swagger.yaml [hunk 4]
```diff
@@ -124,26 +124,20 @@ tags:
...
+      The flow for the API user is as follows:
+      1. Client identifies the Job to which they want to add files. They can create a new Job or get the ID for existing one. As long as it is *not* currently being processed with another Batch)
```
Comment (hchaps) [inline spec/swagger.yaml]: 1. get the ID for existing one > get the ID for an existing one. 
2. there's a stray bracket at the end of the sentence.

File: spec/swagger.yaml [hunk 5]
```diff
@@ -124,26 +124,20 @@ tags:
...
+      2. Client creates a new Batch for the Job to which they want to add files.
+      3. Client  uploads files using the Job Batch Upload endpoint, optionally specifying the target locales list for each file. This endpoint has the same parameters as a generic file upload to a Smartling project for specifying exactly how the file should be handled by Smartling.
```
Comment (hchaps) [inline spec/swagger.yaml]: Client  uploads files (there's an extra space between "client" and "upload".

File: spec/swagger.yaml [hunk 6]
```diff
@@ -124,26 +124,20 @@ tags:
...
+      6. Clients can check the Batch status to understand the status of the individual files that were uploaded.  Batch status will indicate if they were uploaded successfully and provide statistics about the strings for the files added to the Job, or if not, what errors happened while uploading the file.
+      After a Batch is completed it can't be reused.
+      **Limitations for Job Batches**
+      - Correct handling of multiple Batches in parallel is not guaranteed. Creating and working with a new Batch should be done only after previous Batch reached "completed" state. Use the List Batches call on the Job in question to confirm no Batches are currently open (any state other than "Completed").
+      - The entire Smartling system is asynchronous and there is currently no way to "lock" a Job. So this service cannot guarantee correct processing of a Batch it's corresponding Job or the Files that are part of the Batch are modified using the Smartling Dashboard UI or with another integration.
```
Comment (hchaps) [inline spec/swagger.yaml]: Unclear:
So this service cannot guarantee correct processing of a Batch it's corresponding Job or the Files that are part of the Batch are modified using the Smartling Dashboard UI or with another integration.

File: spec/swagger.yaml (hunk 3 above)
Comment (enegron-sl) [reply spec/swagger.yaml]:
```suggestion
      The Job Batches API utilizes internal Smartling events to optimize the process of uploading files and adding them to Jobs.
```

File: spec/swagger.yaml (hunk 6 above)
Comment (enegron-sl) [reply spec/swagger.yaml]: How about: "As such, the Jobs Batch service cannot guarantee the result of processing a Batch will be exactly as expected.  If its corresponding Job or the Files that are part of the Batch have been modified in the Smartling Dashboard UI or with another integration, the results of executing a Batch could be affected."

**Comment by enegron-sl** [comment]
@hchaps Client is used in the context to refer to an application or service or integration. Feel free to change as needed.

File: spec/swagger.yaml (hunk 6 above)
Comment (hchaps) [reply spec/swagger.yaml]: Looks good.

**Pull request closed without merging**